src/index.js

```diff
@@ -1 +1,50 @@
+/* eslint-disable no-console */
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const zlip = require('zlib');
+const path = require('path');
+const { pipeline } = require('stream');
+
+const PORT = 8080;
+
+const server = http.createServer((req, res) => {
```

Comment:
The server does not provide a way for the user to select a compression type. The Brotli compression is used by default.